src/plugins/data/public/plugin.ts

```diff
@@ -18,18 +18,12 @@
  */
 
 import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from '../../../core/public';
```

Comment:
Is this client side plugin still needed since the functionality was removed?

Reply:
Not sure, but as we'll be moving the legacy plugin here pretty soon, I don't think it's worth changing.

Reply:
Just leave a comment in the code. Might prevent someone from getting confused as to why this exists.